zcash_client_sqlite/src/lib.rs

```diff
@@ -675,6 +682,18 @@ impl<'a, P: consensus::Parameters> WalletWrite for DataConnStmtCache<'a, P> {
     }
 }
 
+#[cfg(feature = "transparent-inputs")]
+impl<'a, P: consensus::Parameters> WalletWriteTransparent for DataConnStmtCache<'a, P> {
+    type UtxoRef = UtxoId;
+
+    fn put_received_transparent_utxo(
+        &mut self,
+        output: &WalletTransparentOutput,
+    ) -> Result<Self::UtxoRef, Self::Error> {
+        wallet::put_received_transparent_utxo(self, output)
```

Comment:
I don't see this function being added in this commit; was it previously just being used directly by the mobile SDKs, and not as an inner component of other functions?

Reply:
@nuttycom said yes.